shared/src/commonMain/kotlin/db/database/level/LevelDatabase.kt

```diff
@@ -28,15 +28,13 @@ class LevelDatabase(sqlDriver: SqlDriver) :
             id = id,
             level = request.level,
             requiredPoints = request.requiredPoints,
-            pointsToNextLevel = request.pointsToNextLevel
```

Comment:
What's the reason these are removed?

Reply:
The requiredPoints attribute is how many points are need to reach that level. In the User table we store how many points the user has, so the pointsToNextLevel can be calculated from those 2 attributes, and attributes that can be calculated shouldn't be stored.